lib/irb/workspace.rb

```diff
@@ -107,6 +107,27 @@ def filter_backtrace(bt)
       bt
     end
 
+    def code_around_binding
+      file = @binding.eval('__FILE__')
+      pos  = @binding.eval('__LINE__') - 1
+      return nil unless File.exist?(file)
+
+      if defined?(SCRIPT_LINES__) && SCRIPT_LINES__.key?(file)
+        lines = SCRIPT_LINES__[file].split(/\r?\n/)
+      else
+        lines = File.readlines(file)
```

Comment:
but result strings of `File.readlines` include `\n`.

Which is intentional?
If include `\n`, you can use `lines` instead of `split`.

Reply:
Oops, good catch, thanks https://github.com/ruby/ruby/pull/1764/commits/bb3732f46658ec2ae2e92dbecd84591d351087d5